### PR TITLE
Modify `hs create module` command to correspondence to content_types (Previously host_template_types)

### PR DESCRIPTION
**Pull request by kotto5**
## Description and Context

Modify `hs create module` command's content_type checkbox to choose value for content_types.
Previously, you could choose a value for host_template_types.
It causes an error when you upload modules.

https://github.com/HubSpot/hubspot-cli/issues/1224

## Screenshots

Before
Screenshot 2024-10-27 at 23 38 03

After
Screenshot 2024-10-27 at 23 36 36

After version may have problem about UI. Choices are too many to choose and to display at once so you have to scroll to see all choices. It can make confuse user who create modules.

## TODO

- Discussion about UI and improvement may be needed.

### CODE REVIEW

**Comment by kotto5** [comment]
Is this PR unimportant?

**Comment by kotto5** [comment]
@graysky @also @timmfin @bbeaudreault 
Is this PR unimportant?

File: lib/prompts/createModulePrompt.ts [hunk 1]
```diff
@@ -30,10 +30,18 @@ const CONTENT_TYPES_PROMPT = {
   message: i18n(`${i18nKey}.selectContentType`),
   default: ['PAGE'],
```
Comment (TanyaScales) [inline lib/prompts/createModulePrompt.ts:31]: Hi @kotto5 - one quick note that we should update the default here to point to the `ANY` option instead to support this change

**Comment by TanyaScales** [comment]
@kotto5 - RE: this comment "After version may have problem about UI" and what to do here. I would be fine with this update moving through independently of UI updates to make this better (for the short term). We can discuss the proper pattern for long-form option lists over here on our side and make sure it is consistent across the commands.

File: lib/prompts/createModulePrompt.ts (hunk 1 above)
Comment (kotto5) [reply lib/prompts/createModulePrompt.ts]: Thanks for review @TanyaScales 

I fixed it based on your point, and I'm glad to see that the UI thing is not a problem either.
Hope you will check the fixes.

Comment (kotto5) [reply lib/prompts/createModulePrompt.ts]: @TanyaScales 
How do I ask other reviewers for a review after I have been approved by you?

**Pull request merged**